chaoslib/litmus/disk_fill_by_litmus.yml

```diff
@@ -0,0 +1,161 @@
+- name: Get a Random Pod name
+  shell: >
+    kubectl get pods -l {{ label }} -n {{ namespace }} --no-headers -o custom-columns=:metadata.name | shuf -n 1 
+  args:
+    executable: /bin/bash
+  register: pod_name
+
+- name: Get ephemeral-storage Limit Value
```

Comment:
Also interested in understanding what happens when the ephemeral limits on the pod are not set? How do we plan to handle this. Also, this needs a min kubernetes version (1.13+) to run. 

Can we have a limitations section in the README explaining these ?